### ISSUE TITLE
Rollover Damage

### ISSUE THREAD

**Issue by jay1148** [Cardshifter/Cardshifter#269]
Are we going to have rollover damage, much like Trample from MTG, where any damage left after a unit has done battle goes directly to player life points?

5/5 attacks 3/3 - 5/5 becomes 5/2, owner of 3/3 would take 2 damage directly to life points and unit would die.

**Comment by jay1148** [comment]
I realize this is a fluke post... that units can attack player health points directly. Can we stray away from that. The reason why I is because logically, if I have an army you will have to defeat that army before getting to me... I think that we should have to empty the field before getting to attack the player directly.

**Comment by Zomis** [comment]
Trample is partially implemented already, adding full support for it for specific mods shouldn't be too difficult. It should be fully supported, but is not a high priority at the moment.

**Comment by jay1148** [comment]
So are we unable to attack an opponent directly as long as the opponent has a unit on the table?

**Comment by Zomis** [comment]
Not necessarily. That is the 'taunt' feature. As long as opponent has creatures with taunt in play, you are unable to attack opponent. In Cyborg-Chronicles, all creatures have taunt, but not in Mythos.

**Comment by jay1148** [comment]
I see, and thus lies my issue with it. Logically if I have an army, no mater their attributes you cannot get to me until you defeat them, I feel it should be the same with this. In regards to that, "Taunt" can roughly work the same, instead you are only able to attack a unit with taunt, and no other units. Ultimately, unanimous decision I guess, but that is just my opinion.

**Comment by Zomis** [comment]
I don't see what Taunt has to do with Trample, really. About your concerns with the Taunt functionality and however all units should have that or not, I recommend creating a new issue.

**Comment by Phrancis** [comment]
@jay1148 Cyborg Chronicles has auto-taunt for all units at all times. We're just trying something a bit different, that is, closer to Hearthstone attack rules, with Mythos; hence the different logic. Both can be fun, really, as long as we take care to balance the game accordingly. I say we give this logic a spin for a while and try to make it work. If it really doesn't, then we change it
